lib/shared/server-request/src/request.ts

```diff
@@ -1,3 +1,6 @@
+// NOTE: This file is duplicated in "sdk/js/src/RequestUtils" because nx:rollup cant build non-external dependencies
+// from outside the root directory
```

Comment:
github issue outlining this problem https://github.com/nrwl/nx/issues/10395

Reply:
can you add this as a code-comment.